modules/openideal_idea/openideal_idea.routing.yml

```diff
@@ -14,3 +14,23 @@ openideal_idea.ideas_autocomplete:
     _controller: '\Drupal\openideal_idea\Controller\IdeasAutocompleteController::autocomplete'
   requirements:
     _permission: 'access content'
+
+openideal_idea.openideal_idea_edit_five_stars:
+  path: '/admin/config/openideal_idea/edit-five-stars'
+  defaults:
+    _title: 'Edit five stars widgets'
+    _form: '\Drupal\openideal_idea\Form\OpenidealIdeaEditFiveStarsForm'
+  requirements:
+    _permission: 'access to configuration forms'
+  options:
+    _admin_route: TRUE
```

Comment:
Routes starting with "/admin" will be admin routes by default.
See: https://www.drupal.org/node/2224207